code/modules/clothing/suits/armor.dm

```diff
@@ -320,11 +320,14 @@
 
 /obj/item/clothing/suit/armor/reactive/proc/use_power()
 	if(in_grace_period)
-		return
+		return TRUE
+	if(cell.charge <= 0) //No working if cells are dry
+		return FALSE
 	else
 		in_grace_period = TRUE
 		cell.use(energy_cost) // 20 blocks for most armors, 12 blocks for the "stronger" armors
 		addtimer(VARSET_CALLBACK(src, in_grace_period, FALSE), 1 SECONDS)
+		return TRUE
```

Comment:
I am pretty sure `cell.use()` will return false if there isnt enough power. you could switch this up to early return `if(!cell.use(energy_cost)) => return FALSE`q

also you dont need an else, if the `if()` statement returns

Reply:
Done!